app/models.py

```diff
@@ -21,6 +21,18 @@ class Race(Enum):
     RANDOM = 'Random'
 
 
+class Identity(models.Model):
```

Comment:
can we add `id: uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)`. I just like UUIDs more than incrementing integers